placideirandora/epic-mail
# #164148780 built sign up content

#### What does this PR do?
It adds sign up content to the sign-in-sign-up page.

#### Description of Task to be completed?
1. It adds functionality to the unregistered sign up button so that when it is clicked on, It displays the sign up form.

2. It animates the sign up button by bouncing from the sign in content to the sign up content while having hiddeden the sign in content.

3. When the sign in content is active, The sign up content is hidden. And when the sign up content is active, The sign in content is hidden.

#### How should this be manually tested?
Navigate to the sign-in-sign-up page which displays only the sign in content. Then, Click on the sign up button on the left.  And check if it bounces to the left and displays the sign up content while having hiddeden the sign in content and vice versa.
